custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_1_3.sql

```diff
@@ -0,0 +1,18 @@
+UPDATE "agg_child_health_{start_date}" agg_child_health
+  SET
+    wasting_moderate = ut.wasting_moderate,
+    wasting_severe = ut.wasting_severe,
+    wasting_normal = ut.wasting_normal,
+    stunting_moderate = ut.stunting_moderate,
+    stunting_severe = ut.stunting_severe,
+    stunting_normal = ut.stunting_normal,
+    height_measured_in_month = ut.height_measured_in_month,
+    weighed_and_height_measured_in_month = ut.weighed_and_height_measured_in_month
+  FROM temp_chm_local ut
+  WHERE (
+    agg_child_health.awc_id=ut.awc_id and 
+    agg_child_health.month=ut.month and
+    agg_child_health.age_tranche=ut.age_tranche and
+    agg_child_health.caste=ut.caste and
```

Comment:
The agg level 5 group is wrong here. it also include other fields as well https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py#L185